### PR TITLE
Enabling tests and adapting fixtures for version history

### PR DESCRIPTION
**Pull request by danrot**
Created because #277 had wrong base branch

This PR is still based on #272

### CODE REVIEW

**Comment by dbu** [comment]
can you please rebase this on the versioning branch now so we can see what is new? there seems to be a conflict as well, unfortunately.

**Comment by danrot** [comment]
@dbu done

**Comment by danrot** [comment]
@dbu I can see that there are some changes because of the auto formatter in IntelliJ... Should I try to revert them?

File: composer.json [hunk 1]
```diff
@@ -39,5 +39,11 @@
         "branch-alias": {
             "dev-master": "1.2-dev"
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/danrot/jackalope.git"
```
Comment (dbu) [inline composer.json]: this can go away when https://github.com/jackalope/jackalope/pull/286 is merged, right?

**Comment by dbu** [comment]
hm, that autoformatter seems to have a different code style then we where using. if you could undo the style changes that would help - its hard to see here what really changed.

File: composer.json (hunk 1 above)
Comment (danrot) [inline composer.json]: exactly

Comment (dbu) [inline composer.json]: and that PR is merged, can you revert the changes on this file please?

**Comment by danrot** [comment]
@dbu Now it should be a lot better :-)
